src/components/nodes-table/ColumnFilterNumber.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-card-text>
+    <v-row>
+      <v-col>
+        <v-text-field
+          type="number"
+          label="Min"
+          v-model="value.min"
+          clearable
+          @change="$emit('change', value)"
+        ></v-text-field>
+      </v-col>
+      <v-col>
+        <v-text-field
+          type="number"
+          label="Max"
+          v-model="value.max"
```

Comment:
check that `min` is less then max ? Or at least show a validation error?

Reply:
Done.